src/app/child-dev-project/notes/dashboard-widgets/important-notes-dashboard/important-notes-dashboard.component.ts

```diff
@@ -12,14 +12,14 @@ import { concat, Observable } from "rxjs";
 import { first, map } from "rxjs/operators";
 import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
 
-@DynamicComponent("ImportantNotesComponent")
+@DynamicComponent("ImportantNotesDashboard")
```

Comment:
```suggestion
@DynamicComponent("ImportantNotesDashboard")
@DynamicComponent("ImportantNotesComponent")
```
Or we have to migrate

Reply:
:+1: good point. Adding this into the existing commit as well as into the detectLegacyConfig logging

Reply:
hmm, actually automatic migration would be much more feasible then ... :see_no_evil: